source/develop/release-management/features/network/copy-host-networking.html.md

```diff
@@ -0,0 +1,128 @@
+the administrator can select from which host (`Host0`) should the configuration be copied.
+
+Via REST API every host has method called `copynetworks` which needs a single
+parameter, id of the host (`Host0`) that the configuration should be copied from. 
+
+### Limitation
+
+* The host configuration can be copied only within the same cluster. 
+
+* Configuration of static IP address for any of the networks cannot be copied. The static IP
+networks have their IP incremented by one to stay in range of the mask. 
+
+* Host with same interface names but different physical network connections will result in bad configuration. 
+See more details in [Implementation](#implementation).
+
+* Destination host has to have same amount or more interfaces than host that we are copying from. In case
+of opposite, we cannot determine if we should create bond with missing interface or skip the bond creation
+and use it only for simple network attachment.
+
+
+### Implementation
+
+A new command `CopyNetworks` is added to the engine. That has destination and source host as parameters.
+Both hosts are validated that they belong to the same cluster. And interface count is validated whether
+the destination to has the same number of interfaces or more. Command retrieves interface 
+list from both hosts and order it by name. This ordering might result in wrong network configuration if
+interface names on both hosts are identical but physical connection in data center is different.
+The command then proceeds to preparation of network attachment definitions. Let's refer to host that is
+being to copied as `SourceHost` and host the will receive the new configuration as `DestinationHost`.
+
+* Retrieve list of all interfaces without bonds from `SourceHost` and all interfaces from `DestinationHost`. Ordering
+them by their name. 
+
+* All bond slaves from `SourceHost` are paired with their corresponding partners. New bonds definitions are 
+created accordingly in configuration that will be passed to `DestinationHost`.
+
+* Retrieve list of all interfaces with bonds from `SourceHost`. Exclude bond slaves from both lists and shorten 
+the `DestinationHost` list to match the same length as `SourceHost` list. 
+
+* Copy network attachments for all interfaces in `SourceHost` list. 
+
+* Walk through the new network attachments list and update any static IP address configuration. IP address
+is incremented by one with address check (same subnet and address is not broadcast or network). 
+
+* Send `HostSetupNetworks` with new attachments and bonds for `DestinationHost`. This will ensure that everything
+is done in single transaction and configuration will be reverted on any possible error. 
+
+New button and dialog is added to Webadmin UI. Button is called `Copy Host Networking` and upon click opens
+new dialog window that let administrator select another host from the same cluster to copy the configuration 
```

Comment:
Can this be implemented as UI plugin?

Reply:
It probably can. Because we will have REST API endpoint anyway.

Reply:
Maybe we can address this in a second PR: Can we discuss the new UI with Laura?